palette/support/src/weights.rs

```diff
@@ -210,6 +208,20 @@ impl<T> ClassifyDispatch<T> for SimpleDispatchInfo {
 	}
 }
 
+impl PaysFee for SimpleDispatchInfo {
```

Comment:
Okay but I don't think this is enough for your use case right? I would allow a way to express this through the annotation (similar to https://github.com/paritytech/substrate/pull/4158, or we can just wait for that). Technically you want to have a, lets say operational, transaction with weight 100, once with fees and once without. I think this simple Enum is no longer enough and we need something better, like https://github.com/paritytech/substrate/pull/4158

Reply:
Maybe I should have `FreeNormal` and `FreeOperational` takes a weight? Does it make sense to actually have a weightless transaction?

Reply:
At the simplest, you can make all of them get a `bool` that gets mapped to pays_fee. So you can say: `FreeOperation(true)` and `FreeOperational(false)`. But I think this is no longer _Simple_ stuff as the name says.  I think we can ditch SimpleDispatchInfo entirely and come up with better stuff. I will think about it.

Reply:
Yeah. I think this can be done in later PR.

Reply:
sure, will make an issue for it.